### PR TITLE
Update the glossary.md TPS definition

### PR DESCRIPTION
**Pull request by qwahzi**
Per Discord discussion: updating the historical TPS definition to be more accurate

### CODE REVIEW

File: docs/glossary.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ The [account](#account) if the block is the first block on the account, otherwis
...
+Often used to refer to the rate of complete transactions between two parties (i.e. a send with a corresponding receive). In the past, TPS was a per-node measurement that represented the perceived network-level transmission rate ([BPS](#blocks-per-second-bps)), but this measurement was found to be somewhat inaccurate due to peering and propagation differences between nodes. TPS is now used to refer to ([Confirmations Per Second](#confirmations-per-second-cps)/2) which is more similar to the TPS metric used by other cryptocurrencies (e.g. Bitcoin). Nano transactions do not require a corresponding receive to be considered [confirmed](#confirmation), but a receive block does need to be confirmed before the received funds can be spent again (see [pending](#pending)).
```
[the remark this answers is not in the document]
Comment (qwahzi) [reply docs/glossary.md]: My only concern would be a potentially ambiguous them (sends, receives, both?). What about this?

> Nano sends do not require a corresponding receive to be [confirmed](#confirmation), but a receive block...

[the remark this answers is not in the document]
Comment (zhyatt) [reply docs/glossary.md]: Sounds good!

[the remark this answers is not in the document]
Comment (qwahzi) [reply docs/glossary.md]: https://github.com/nanocurrency/nano-docs/pull/122/commits/56354d15bca121bcd6e63bd53409700ad984808f

**Comment by zhyatt** [comment]
Thanks @qwahzi !

**Pull request merged**